handbook/engineering/testing.md

```diff
@@ -0,0 +1,9 @@
+# Testing
+
+This file documents how we test code at Sourcegraph. See [permissions](#permissions) for help on accessing our testing tools. 
```

Comment:
I don't think we need links within the document like this, especially when the document is so small.

```suggestion
This file documents how we test code at Sourcegraph.
```